library/package.json

```diff
@@ -49,22 +49,31 @@
     "eslint.fix": "eslint \"src/**/*.ts*\" --fix",
     "format": "prettier --write ./src",
     "format.check": "prettier --check ./src",
+    "babel.regex": "babel src/regex.ts --out-file transpiled/regex.ts",
+    "prebuild": "pnpm babel.regex",
     "build": "tsup",
+    "postbuild": "rimraf transpiled",
```

Comment:
Is `rimraf` needed here? What is the disadvantage of `rm -rf`?

Reply:
I added `rimraf` because of concerns about `rm -rf` working cross-platform, but I'm not sure where this would actually be a problem so I'll go ahead and replace it with `rm -rf`.